components/Footer/Footer.js

```diff
@@ -1,14 +1,18 @@
 import styled from "styled-components";
 import Link from "next/link";
 import HomePageIcon from "../../public/icons/home_page";
+import UserIcon from "../../public/icons/user";
 
 export default function Footer() {
   return (
     <footer>
       <StyledNav>
-        <StyledLink href="/">
+        <StyledLink href="/" aria-label="to home page">
           <HomePageIcon color={`var(--red-vine-color)`} size="30px" />
         </StyledLink>
+        <StyledLink href="/my-data" aria-label="to my data">
+          <UserIcon color={`var(--red-vine-color)`} size="30px" />
```

Comment:
If you have several icon-components that are quite similar in styling, why not making one component and pass down attributes? src, alt, color etc...?

Reply:
You are right. I change the color and size in Icon components